admin/javascript/SSNotice.js

```diff
@@ -0,0 +1,205 @@
+/**
+* File: SSNotice.js
+*
+* Replace the old statusMessage/errorMessage
+* with a custom library: ss.notice
+*
+* Allows for concistent notices/message display throughout the CMS
+* via simple JavaScript function calls:
+* - ss.notice.message(text, [options]);
+* - ss.notice.success(text, [options]);
+* - ss.notice.warning(text, [details], [options]);
+* - ss.notice.error(text, [details], [options]);
+**/
+(function($, root) {
+
+	// local working version of the library object
+	var lib = {};
+
+	/**
+	 * Library settings
+	 */
+	lib.settings = {
+		parentSelector: '.cms-content-fields',
+		$notice: $('<p class="message"/>'),
```

Comment:
If we are inserting text into existing element we would have to duplicate those anyway to create new one etc...

Am all for moving the html to a template. Thought this could be done in a JS template a bit like https://github.com/silverstripe/silverstripe-framework/blob/3.2/javascript/UploadField_uploadtemplate.js so the template can easily be chosen when calling the JS function or overridden by dev if needed. How does that sound?